features/telemetry/protocols/netflow/transport/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/common/FlowMessage.java

```diff
@@ -0,0 +1,266 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2020 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2020 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.telemetry.protocols.netflow.common;
+
+import org.opennms.netmgt.telemetry.protocols.netflow.proto.Direction;
+import org.opennms.netmgt.telemetry.protocols.netflow.proto.NetflowVersion;
+import org.opennms.netmgt.telemetry.protocols.netflow.proto.SamplingAlgorithm;
+
+import com.google.protobuf.DoubleValue;
+import com.google.protobuf.UInt32Value;
+import com.google.protobuf.UInt64Value;
+
+public interface FlowMessage {
```

Comment:
I don't think we need this interface at all. Instead, we should use the `FlowMessageProto` directly in the converters.